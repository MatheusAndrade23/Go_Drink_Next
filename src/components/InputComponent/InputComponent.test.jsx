import { screen } from '@testing-library/react';
import { renderTheme } from '../../tests/render-base.js';
import { InputComponent } from '.';

const mockFunction = () => {};

describe('<InputComponent />', () => {
  it('should render', () => {
    const { container } = renderTheme(
      <InputComponent
        placeholder="Something"
        name="test"
        handleChange={mockFunction}
      />,
    );
    expect(container).toMatchSnapshot();
  });
});
