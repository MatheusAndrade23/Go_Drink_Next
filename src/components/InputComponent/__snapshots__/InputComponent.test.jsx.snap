// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputComponent /> should render 1`] = `
.c0 {
  width: 300px;
  outline: none;
  padding: 8px;
  border: 1px solid #00ADB5;
  border-radius: 8px;
}

<div>
  <input
    class="c0"
    name="test"
    placeholder="Something"
    title="Something"
    type="text"
  />
</div>
`;
